example/generator/odp_generator.c

```diff
@@ -25,6 +25,8 @@
 #define MAX_WORKERS            (ODP_THREAD_COUNT_MAX - 1)
 #define POOL_NUM_PKT           2048  /* Number of packets in packet pool */
 #define POOL_PKT_LEN           1856  /* Max packet length */
+#define MAX_HDR_LEN            128   /* L2/L3/L4 header length */
+#define POOL_SEG_LEN           1856  /* Segment length to pool create */
```

Comment:
Why 1856? It seems MAX_HDR_LEN would be sufficient.

Reply:
Yes it will works with MAX_HDR_LEN, 1856 (i.e., POOL_PKT_LEN) will avoids multi segments for regular MTU and gives better performance.